assemblers/org.osbuild.tar

```diff
@@ -40,16 +40,17 @@ def main(tree, output_dir, options):
     filename = options["filename"]
     compression = options.get("compression")
 
-    command = ["tar", "-cf", f"{output_dir}/{filename}", "-C", tree]
-
+    extra_args = []
     if compression is not None:
         if compression not in {"bzip2", "xz", "lzip", "lzma", "lzop", "gzip"}:
             return 1
-        command.append(f"--{compression}")
+        extra_args.append(f"--{compression}")
+
 
-    command.append(".")
+    subprocess.run(["ls", "-l", "/usr/bin"])
```

Comment:
What's this?

Reply:
This was me wondering if `tar` is in the build root. (It wasn't)

Fixed.